paperqa/agents/__init__.py

```diff
@@ -67,13 +67,16 @@ def configure_cli_logging(verbosity: int = 0) -> None:
 
     rich_handler.setFormatter(logging.Formatter("%(message)s", datefmt="[%X]"))
 
+    module_logger = logging.getLogger("paperqa")
```

Comment:
Can we import this somehow?

Reply:
yea I think we can just split __name__ actually